devimint/Cargo.toml

```diff
@@ -34,6 +34,7 @@ fedimint-server = { path = "../fedimint-server" }
 fedimint-testing = { path = "../fedimint-testing" }
 futures = { workspace = true }
 hex = { workspace = true }
+lazy_static = "1.4.0"
```

Comment:
BTW. We should stop using lazy_static

https://morestina.net/blog/2055/rust-global-variables-two-years-on

Reply:
Should we create an issue to remove existing usage and add a semgrep to block it going forward?

Reply:
@tvolk131 :+1:

Reply:
Neat!